pyproject.toml

```diff
@@ -30,7 +30,7 @@ exclude = '''
 line_length = 89
 profile = "black"
 skip_gitignore = true
-known_first_party = ["bluemira"]
+known_first_party = ["bluemira", "EUDEMO_builders"]
```

Comment:
As you're fixing this "EUDEMO_tests" here too? eg if there are any imports between the tests

Reply:
I can do - we don't do this for the `bluemira` tests, though.

Reply:
Fair point, I see you've changed it now either way. Leave it changed for now and we can remove it when this is a separate repo